app/views/users/edit.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1><%= t(".title", name: user.name) %></h1>
+<%= simple_form_for user do |f| %>
+  <%= f.input :name %>
+  <%= f.input :email %>
+  <%= f.submit t(".submit") %>
+<% end %>
+
+<%= link_to t(".delete"), user, :method => :delete %>
```

Comment:
A suggestion to use the new hash syntax: `method: :delete`?

Reply:
Thanks Elle, good pickup, I want to be as consistent as possible.